src/Support/Utility.php

```diff
@@ -205,8 +205,8 @@ public static function generateQuery($query = array())
             $query->_query['environment'] = $query->contentType
                 ->stack->getEnvironment();
             $subQuery = array();
-            if (count($query->subQuery) > 0) {
-                $subQuery['query'] = json_encode($query->subQuery);
+            if (isset($query->subQuery)) {
+                $subQuery['query'] = $query->subQuery;
```

Comment:
@mcarcangiu-ea , 
The change for `count` to `isset` is working on our end. 
But, the reason we have `json_encode` is to convert the query for example:

`$Stack->ContentType(CT_ContentType)->Query()->toJSON()->where('title', 'CB1-1')->find()`

this will need to be passed in URL parameter in following pattern:

`environment=mobile&query=%7B%22title%22%3A%22CB1-1%22%7D`

but the change you suggested is converting it to:

`environment=mobile&query%5Btitle%5D=CB1-1`

this will affect all the other SDK user.

Can you please check and update if these condition (ex. Query any Content Type to find entries) are working properly on you end. If not working please update the PR request.